recipe/migrations/libgrpc151.yaml

```
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+libgrpc:
+- '1.51'
```

If we want we could do one last migration that also updates `grpc_cpp`. I'm planning to remove that output for 1.52, so then it would remain frozen at that point. Or we leave it frozen at 1.49 directly.
```suggestion
grpc_cpp:
- '1.51'
libgrpc:
- '1.51'
```

---

I'm somewhat confused. What do you want? To hold off until we build some critical packages or merge this?

---

No, we should merge the PR, the only question is if we want to migrate both the legacy output (`grpc_cpp`) or only the new name (`libgrpc`). 

Depending on whether you accept the above diff, the global cbc (upon closing of the migration!) would look like:

A) after migrating just `libgrpc`
```
# legacy output, keep it so old recipes don't accidentally become unpinned
grpc_cpp:
- '1.49'
[...]
libgrpc:
- '1.51'
```

B) after migrating both:
```
# legacy output, will not advance past 1.51
grpc_cpp:
- '1.51'
[...]
libgrpc:
- '1.51'
```

---

If anything is still unclear, you can just forget about it and push merge 😆

---

> both the legacy output (grpc_cpp) or only the new name (libgrpc)

if we want to cleanup, we should do it in a concerted effort. For now, lets keep them in sync.

---

I already did replace all the usages of `grpc-cpp` I could find, but fine by me.